src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -373,6 +373,9 @@
    :ui/toggle-right-sidebar         {:binding "t r"
                                      :fn      ui-handler/toggle-right-sidebar!}
 
+   :ui/close-right-sidebar-top     {:binding "d r"
```

Comment:
Two issues with this:
1. This is a new letter for key combos. We only have 3 other letters like this - (t)oggle, (n)ew and (g)o. Since we want to minimize breaking changes, when we add new key combos we should do so with the intent of using that letter for multiple future commands
2. This key combo does not start the same as the command unlike all our other key combos e.g. (t)oggle. This could be resolved by changing the shortcut to "c r" or changing the command name to "Delete ..."

Personally I don't see the motivation to introduce a new key combo for one command but don't have a strong opinion as long as the 2nd issue is addressed. Users can always add their own bindings in their config. What do you think @andelf @cnrpman @sprocketc @CakeCrusher?

Reply:
I agree with @logseq-cldwalker.

If we decide to include this, I would also change the namespace to `:sidebar/`. We already have `:sidebar/clear`, so I would probably use `:sidebar/clear-top`, and the shortcut would be `c t`. `c r` might be misleading. The only inconsistency in this case would be the existing shortcut of `:sidebar/clear`, which is `mod+c mod+c`, but I am not sure if we should change that.

Reply:
@sprocketc @logseq-cldwalker 
I agree with both suggestions. Unfortunately the binding `c` for `:git/commit` would override all `c...` commands. Considering how narrow that command was, I changed it to `n c` for "new commit".

Reply:
@CakeCrusher The :git/commit shortcut is going to be changed

Reply:
@cnrpman got it. Reverted it to the old state.